### DISCUSSION TITLE
Combination of Quantconv3d and QuantConvTranspose2d

### DISCUSSION THREAD

**Discussion by ChristiaanBoe** [Xilinx/finn#879] [General]
@hleblevec @icolbert @auphelia Hey everyone,

I wish to test out the QuantConvTranspose2 added by @hleblevec  and combine it with my own implementation of a Quantconv3d operator. How would I best go about doing this? Are there plans for merging the QuantConvTranspose2  with the dev branch or is it better for me to switch to the branch of @hleblevec?

**Comment by hleblevec** [comment]
Hey @ChristiaanBoe,
The QuantConvTranspose2d support will probably be merged to dev in the future but as for now it is still ongoing as it requires a full end-to-end test case to be done. If you want to use it right away I suggest you switch to the associated branch in my fork repo: https://github.com/hleblevec/finn-fork/tree/feature/pixelpadding, but there might still be some bugs and it is subject to changes. Let me know if you have any questions or if you experience bugs using it.
